node/tests/TestUtilities.ts

```diff
@@ -607,13 +607,22 @@ export async function encodableTransactionTest(
  */
 export async function encodedTransactionTest(
     baseTransaction: Transaction | ClusterTransaction,
+    version: string,
 ): Promise<[string, GlideReturnType][]> {
     const key1 = "{key}" + uuidv4(); // string
     const key2 = "{key}" + uuidv4(); // string
     const key = "dumpKey";
-    const dumpResult = Buffer.from([
-        0, 5, 118, 97, 108, 117, 101, 11, 0, 232, 41, 124, 75, 60, 53, 114, 231,
-    ]);
+    let dumpResult: Buffer;
+
+    if (gte(version, "7.0.0")) {
+        dumpResult = Buffer.from([
+            0, 5, 118, 97, 108, 117, 101, 11, 0, 232, 41, 124, 75, 60, 53, 114,
+            231,
+        ]);
+    } else {
+        dumpResult = Buffer.from([118, 97, 108, 117, 101]);
+    }
```

Comment:
Please remove this. Checking binary dump in conditions where format may change isn't a good idea.
It is better to validate that a key could be successfully restored using that dump with the original value.
Same applicable to line 670.

I'd move part relevant to dump-restore to another test to avoid making this function more complex.
See https://github.com/valkey-io/valkey-glide/blob/e4192299e7964c0dfa66753fe7c37281b4870b1e/java/integTest/src/test/java/glide/standalone/TransactionTests.java#L691-L711 for reference